<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;DayOfWeek</string>
   </property>
   <property name="buddy">
    <cstring>lineEditDayOfWeek</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>何夕(&amp;H)</string>
   </property>
   <property name="buddy">
    <cstring>lineEditHeXi</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDayOfWeek">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHeXi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Year</string>
   </property>
   <property name="buddy">
    <cstring>lineEditYear</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
